schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/activity/ActivityType.java

```diff
@@ -79,7 +80,7 @@ public String getName() {
 
     public void setName(String name) {
         this.name = name;
-        this.lcName = name;
+        setLcName(StringUtils.lowerCase(name));
```

Comment:
the lowerCase here is no harm, but as an FYI we use `LowerCaseListener` so when it is persisted it get lowercased...

```java
    @Attribute(listener = LowerCaseListener.class)
    private String lcName;
```